_episodes/21-carpentries.md

```diff
@@ -4,7 +4,8 @@ block: "The Carpentries"
 teaching: 30
 exercises: 45
 questions:
-- "How are Software, Data, and Library Carpentry organized and run?"
+- "How is The Carpentries organized and run?"
+- "What is the difference between SWC, DC, and SWC workshops?"
```

Comment:
Should read "SWC, DC, and LC workshops?"

Reply:
good catch!